### ISSUE TITLE
Errors in mwe annotations

### ISSUE THREAD

**Issue by sebschu** [UniversalDependencies/UD_English#13]
(Reported by Bruno Guillaume via email.)

Some mwe relations are strange:
The link : http://talc2.loria.fr/grew/?custom=57f65eebaf73e&corpus=UD_English-1.3 give 6 occurrences of the following pattern:
  * a mwe link from N1 to N2
  * a node X on the right of N1, not linked to N1 with a mwe

**Comment by nschneid** [comment]
I notice a couple of these involve typos: "do to" should be "due to", "becuse of" should be "because of". Can the lemmas be edited to reflect the normal spelling?

**Comment by nschneid** [comment]
With the current version of the corpus there is [one `fixed` expression with an intervening word](http://match.grew.fr/?corpus=UD_English-EWT@dev&custom=61f4c169815fb): "due largely to"

**Comment by amir-zeldes** [comment]
I don't think that `fixed` has to always be contiguous - especially in languages that have Wackernagel particles or similar items that can interrupt any phrase, I think fixed expressions should retain their analysis even around those interruptions. 

I don't see a real syntactic difference between the dependency structure of "in large part due to X" and "due in large part to X" - so I would like them to have the same graph structure, notwithstanding the word order.

**Comment by nschneid** [comment]
Agreed that we should not have two different analyses for "due" + "to". But in English, it seems that `fixed` is intended for completely frozen expressions ["that behave like a single function word"](https://universaldependencies.org/en/dep/fixed.html). So maybe this is evidence against `fixed` for "due to", and similarly for "owing to", "according to", etc.

One test could be interruptability with "not" and coordination:
- The mission's failure was due not to human error but to unforeseeable circumstances. (this was my example but COCA has 61 matches of "due not to")
- "where they would be treated according not to color but to caliber" (COCA)

Anyway this seems like it calls for further discussion on the docs issue tracker or in a meeting.

**Comment by amir-zeldes** [comment]
OK, if we're talking in general about whether "due to" should be fixed then that's another discussion of course, and I'm happy to talk about it next time; but in general I suspect many of the current `fixed` list are not 100% unmodifiable, so if it's a question of quantities, then "due to" is certainly overwhelmingly more common in its uninterrupted form. 

I guess I've just come to accept the current English `fixed` list as a fait accompli in the interest of stability, but it's not something I feel has deep theoretical motivation either.

**Comment by nschneid** [comment]
This has become more urgent as the validator now prohibits `fixed` with intervening words.

I wonder if we should establish a policy that simply excludes from the fixed list all ADJ+ADP combinations acting somewhat like adpositions. That would mean removing "due to" and "prior to", which are currently in the list, and would resolve the question of whether "other than" should be added (#431). This is assuming we can come up with an acceptable non-`fixed` analysis of the fronted ones ("Due largely to the hot weather, people are staying indoors").

**Comment by nschneid** [comment]
> This has become more urgent as the validator now prohibits `fixed` with intervening words.

Never mind, I misunderstood the validator output—it is a warning, not an error. There is no firm prohibition on intervening words in a fixed expression.
